createDockerCommands.py

```diff
@@ -17,6 +17,25 @@
 inner_folds = sys.argv[15]
 currentWorkingDir = os.path.dirname(os.path.realpath(__file__))
 
+
+def path_to_list(path):
```

Comment:
DRY principle--Import these functions instead of copy+pasting them.

Reply:
[Here's a link](https://stackoverflow.com/a/20309473/6481442) with info on how to do that.